doc/internal/man7/DERlib.pod

```diff
@@ -0,0 +1,148 @@
+=pod
+
+=head1 NAME
+
+DERlib - internal OpenSSL DER library
+
+=head1 DESCRIPTION
+
+OpenSSL contains an internal small DER reading and writing library,
+as an alternative to the publically known i2d and d2i functions.  It's
+solely constituted of functions that work as building blocks to create
+more similar functions to encode and decode larger structures.
+
+All these functions have similar function signatures (C<something>
+will vary depending on what the function will encode):
+
+    int DER_w_something(WPACKET *pkt, int cont, ...);
+
+=begin comment
+
+When readers are added, add this:
+
+    int DER_r_something(PACKET *pkt, int cont, ...);
+
+=end comment
+
+I<pkt> is the packet context used, and I<cont> should be the context
+number of the element being handled, or -1 if there is no context
```

Comment:
Should the -1 (all of them?) be in ```B<>```?  I don't know, I'm asking.

Reply:
Not as far as I know. Do you often see numbers given in bold I man-pages?